admin/spec/components/solidus_admin/ui/badge/component_spec.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe SolidusAdmin::UI::Badge::Component, type: :component do
+  it "renders the overview preview" do
+    render_preview(:overview)
```

Comment:
I tried it locally and it's failing with:

```ActionView::MissingTemplate:
       Missing template layouts/solidus_admin/preview with {:locale=>[:en], :formats=>[:html], :variants=>[], :handlers=>[:raw, :erb, :html, :builder, :ruby]}.
     
       Searched in:
         * "/home/solidus_user/app/admin/spec/components/previews"
         * "/home/solidus_user/app/admin/spec/dummy/app/views"
         * "/home/solidus_user/gems/gems/view_component-3.2.0/app/views"
         * "/home/solidus_user/gems/gems/railties-7.0.4.3/lib/rails/templates"
     
       Did you mean?  solidus_admin/ui/badge/component_preview/overview
                      view_components/preview
                      view_components/previews
                      layouts/application
                      rails/info/properties
                      solidus_admin/ui/badge/component_preview
```

For some reason, it looks like CI is only running `core` & `backend` for this PR :thinking: It probably has nothing to do with the work here, so we need to investigate...

Reply:
I tried locally and it works fine, maybe we used different commands?

```
solidus_admin:elia/admin/ui-showcase ⤑ bin/rspec admin/spec/components/solidus_admin/ui/badge/component_spec.rb                                   ~/C/N/solidus_admin
$ cd admin; bundle exec rspec /Users/elia/Code/Nebulab/solidus_admin/admin/spec/components/solidus_admin/ui/badge/component_spec.rb; cd -

Randomized with seed 64336
.

Finished in 0.24157 seconds (files took 2.02 seconds to load)
1 example, 0 failures

Randomized with seed 64336
```

Reply:
> For some reason, it looks like CI is only running `core` & `backend` for this PR 🤔 It probably has nothing to do with the work here, so we need to investigate...

@waiting-for-dev I was fooled by CircleCI as well at first, you can find `admin` in one of the other parallel runs, `bin/build-ci` tries to split the work among parallel runs.

<img width="316" alt="image" src="https://github.com/solidusio/solidus/assets/1051/02e5862a-9f29-47f6-81d1-1a97c7e6ef77">

Reply:
That's really weird. @rainerdema, @kennyadsl, could you try it and confirm who's seeing ghosts here? :slightly_smiling_face: I tried once more, and it fails for me...`q

Reply:
Let me try again!

Reply:
@rainerdema had the same issue, looks like there's a fix for that issue in ViewComponent 3.3, I'll bump the dependency in solidus_admin.gemspec 👍 

Probably me and @kennyadsl already have 3.3.

Reply:
I bumped the dependency to 3.3, but I'm still experiencing the problem. Is it the same for you, @rainerdema? :thinking:

Reply:
@waiting-for-dev the dependency must be updated on the root Gemfile, not the sandbox one, I thought it was worth mentioning since this tricked me at first.

Reply:
@waiting-for-dev I can confirm that I was experiencing the same exception yesterday.
But after I switched from version 3.2.0 to 3.3.0, the error ceased to occur 🤔

The fix for the problem released in 3.3.0 seems to be this specifically:
https://github.com/ViewComponent/view_component/pull/1777

Reply:
Bumping to v3.3.0 alone didn't fix my issue, but after re-generating my `Gemfile.lock` file the issue is gone.